File: Scripts/laptop/url-encoding/php/url-encoding.php
#!/usr.bin/php 

<?php
$r = 1;

function executeTask($i) {

	$s = 'http://foo/bar/';
	$s = rawurlencode($s);
	return $i + 1;
}

for ( $i = 0; $i < 10000000; $i++) {
	$r = executeTask($i + $r);
}
?>

File: Scripts/laptop/url-encoding/go/url-encoding.go
package main

import (
 //   "fmt"
    "net/url"
)

func executeTask(i int) int {
	
    url.QueryEscape("http://foo bar/")
	return i + 1
}

func main() {
  r := int(1)
  for i := 0; i < 100000000; i++ {
  	r = executeTask(i + r)
  }
}
